Issues with plugins on Hyper 1.2.0 & 1.2.1

- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate


- **OS version and name**: `10.12.3 macOS Sierra`
- **Hyper.app version**: `1.2.1`
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: `Default`
- **Relevant information from devtools** _(CMD+ALT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: `N/A`
- **The issue is reproducible in vanilla Hyper.app**:  `Is Vanilla`

## Issue
Getting `Error updating plugins.` even without any plugins. It says to check for `~/.hyper_plugins/npm-debug.log` but no such file exists.

Seems to be having issues with downloading plugins as well. Getting `Plugin "name" failed to load (Cannot find module '/Users/henrik/.hyper_plugins/node_modules/name')`

Tried installing fresh, simply reinstalling as well. Still remains.

--- a/app/plugins.js
+++ b/app/plugins.js
@@ -241,7 +241,7 @@ function install(fn) {
       posix: 'npm prune && npm install --production'
     };
     // determine the shell we're running in
-    const whichShell = cfgShell.match(/fish/) ? 'fish' : 'posix';
+    const whichShell = (typeof cfgShell === 'string' && cfgShell.match(/fish/)) ? 'fish' : 'posix';
     // Use the install command that is appropriate for our shell
     exec(installCommands[whichShell], {
       cwd: path,

--- a/app/plugins.js
+++ b/app/plugins.js
@@ -241,7 +241,7 @@ function install(fn) {
       posix: 'npm prune && npm install --production'
     };
     // determine the shell we're running in
-    const whichShell = shell.match(/fish/) ? 'fish' : 'posix';
+    const whichShell = cfgShell.match(/fish/) ? 'fish' : 'posix';
     // Use the install command that is appropriate for our shell
     exec(installCommands[whichShell], {
       cwd: path,

